client_test.go

```diff
@@ -31,3 +31,21 @@ func TestClientGeneratePayment(t *testing.T) {
 	}
 	fmt.Println(*response.PgRedirectUrl)
 }
+
+func TestClientGetPaymentStatus(t *testing.T) {
+	client, err := NewClient("https://api.paybox.money/", "542185", "7wnP659u3v9J8jbG", "test")
```

Comment:
@beksmo31 is this good? you just exposed credentials 😄 xaxa

Reply:
lol ))